cmd/guacingest/cmd/ingest.go

```diff
@@ -45,6 +46,7 @@ func ingest(cmd *cobra.Command, args []string) {
 		viper.GetString("nats-addr"),
 		viper.GetString("csub-addr"),
 		viper.GetString("gql-addr"),
+		viper.GetBool("async-ingest"),
```

Comment:
Looks like this arg is not being used now?

Reply:
looks like a rebase issue?

Reply:
@jeffmendoza @pxp928 yes, sorry my mistake. I removed the problematic commit.